### PR TITLE
feat: address bar

### LINKED ISSUE #5: Implement Address Bar and User Modal Components
## Description
Develop a fully functional Address Bar component as per the design provided in the [FIGMA FILE](https://www.figma.com/design/pL9HcD5saTR6TtMsZYyS7z/STARKNET-UI-KITS?node-id=0-1&node-type=canvas&t=gT1lFVCNijoXRGlC-0) along with all it’s variants. This component shows the connect user’s [starknet.id](http://starknet.id/) name and pfp or address, and pops up a user modal on click.

## Requirements

- Design: Follow the provided Figma design precisely for both light and dark mode. Ensure that the component is fully responsive across different screen sizes.
- Style Guide: Adhere strictly to the project’s existing style guide for consistency in UI and UX.
- Functionality: The component should be fully functional, allowing users to seamlessly switch networks.

After building the component, create a sub-route in the `preview` route for the component and display each of it’s variant for easy testing. 

## Guidance
There is an already implemented Connect Button component which can serve as a reference for styling, functionality and previewing.

## Note
No other external dependencies should be installed other than the ones already provided.

### PR DESCRIPTION
**Pull request by Gift-Naomi**
Resolves #5 
Screenshot 2024-10-03 162653
Screenshot 2024-10-03 162505
Screenshot 2024-10-03 162858
Screenshot 2024-10-03 163101

### CODE REVIEW

**Comment by stephanniegb** [comment]
Hey, deployment fails
Screenshot 2024-10-03 at 5 31 15 PM

File: public/svg/Check.tsx
Comment (stephanniegb) [inline public/svg/Check.tsx:1]: Use lucide-react for icons as much as possible eg for the copy, close and check

File: app/components/CopyButton.tsx
Comment (stephanniegb) [inline app/components/CopyButton.tsx:1]: Please use a library for copy functionality

**Review by stephanniegb — CHANGES_REQUESTED** [review]

**Comment by stephanniegb** [comment]
Screenshot 2024-10-06 at 6 39 17 PM
Just delete the pnpm file

**Comment by Oshioke-Salaki** [comment]
lgtm 🚀

**Pull request merged**